rcfc/server.py

```diff
@@ -101,6 +101,16 @@ def get_buttons_registered():
     return {"buttons": buttons_with_state}
 
 
+@get("/groups")
```

Comment:
I faced a challenge in the javascript which might need more changes in this code. When making the buttons for the 'drop down' menu (that doesn't actually work), I found out that id's can't have spaces, which means I probably need to add some sort of 'group id' functionality here, and return that with 'groups'.